<launch>
  <node name="under_carriage_4wheel" pkg="harurobo2022" type="under_carriage_4wheel" output="screen" />
  <node name="can_subscriber_node" pkg="harurobo2022" type="can_subscriber_node" output="screen" />
  <node name="emergency_controller" pkg="harurobo2022" type="emergency_controller" output="screen" />
  <node name="slcan_bridge" pkg="can_plugins" type="slcan_bridge" output="screen" />
  <node name="joy_node" pkg="joy" type="joy_node" output="screen" />
</launch>